src/components/Transaction.vue

```diff
@@ -198,6 +200,11 @@ async function confirm() {
     let transaction_id: string;
     isTransacting.value = true;
 
+    // Apply edits from the Code component
+    if (codeComponent.value && typeof codeComponent.value.applyEdit === 'function') {
+        await codeComponent.value.applyEdit();
+    }
```

Comment:
If transaction contains an error it will not create an error modal and will continue with an unedited transaction. We ideally should show an error modal and keep the edit window active